networkit/cpp/io/NetworkitBinaryWriter.cpp

```diff
@@ -0,0 +1,201 @@
+/*
+ * NetworkitBinaryWriter.cpp
+ *
+ *@author Charmaine Ndolo <charmaine.ndolo@b-tu.de>
+ */
+
+#include <networkit/io/NetworkitBinaryWriter.hpp>
+
+namespace NetworKit {
+
+NetworkitBinaryWriter::NetworkitBinaryWriter(uint64_t chunks): chunks(chunks) {}
+
+uint64_t NetworkitBinaryWriter::splitNodes(uint64_t nodes, uint64_t chunks) {
+
+	nodesPerChunk = nodes / chunks;
+	return nodesPerChunk;
```

Comment:
This is quite confusing: the function both sets a member variable and returns a value. Make it either only compute the number of nodes per chunk, or do not write `nodesPerChunk` inside this function.

Reply:
See below for a suggestion to remove `splitNodes` entirely.